plugin/swift/ShareExtensionViewController.swift

```diff
@@ -199,7 +199,17 @@ class ShareExtensionViewController: UIViewController {
           provider.loadItem(forTypeIdentifier: kUTTypeURL as String, options: nil) { (urlItem, error) in
             DispatchQueue.main.async {
               if let sharedURL = urlItem as? URL {
-                sharedItems["url"] = sharedURL.absoluteString
+                if sharedURL.isFileURL {
```

Comment:
I think it'd be cleaner to handle this via `hasItemConformingToTypeIdentifier` conforming to `kUTTypeFileURL`. I saw that you tried this in the first commit, did you run into any issues with it?

Reply:
files never came through `kUTTypeFileURL`, even after trying the changes on a physical device. i used patch package and tested it in prod using my app Linkvite, if you were wondering.

files only came through `kUTTypeURL` no matter where i tried sharing them from. i don't mind going back to the initial commit but i feel like it might be dead code.

Reply:
okay cool, then let's keep it like this, without `kUTTypeFileURL`